src/main/java/model/UserRequestConverter.java

```diff
@@ -1,7 +1,134 @@
+    	else if(option.contains("-d")) regx_brith = "^\\d{2}$";
+    	return Pattern.matches(regx_brith, str);
+    }
+    
+    public static boolean isValidcertiTrue(String str) {
+    	return str.contains("PRO") || str.contains("ADV") || str.contains("EX");
+    }
+    
+    public static boolean isValidemployeeNumTrue(String str) {
+    	String regx_employnum = "^\\d{8}$";
+    	return Pattern.matches(regx_employnum, str);
+    }
+    
+    public static boolean isValidAddArgsTrue(String[] str) {
+    	return isValidemployeeNumTrue(str[4]) && isValidnameTrue(str[5],str[2]) && isValidclTrue(str[6])
+    			&& isValidphoneNumTrue(str[7],str[2]) && isValidbirthdaytTrue(str[8],str[2]) && isValidcertiTrue(str[9]);
+    }
+
+    public static boolean isValidFirstArgsTrue(String str) {
+    	if(str.contains("employeeNum")||str.contains("name")||str.contains("cl")||
+    			str.contains("phoneNum")||str.contains("birth")||str.contains("certi")) return true;
+    	return false;
+    }
+
+    public static boolean isValidSecondArgsTrue(String str1,String str2, String option) {
+
+    	if(str1.contains("employeeNum")) return isValidemployeeNumTrue(str2);
+    	else if(str1.contains("name")) return isValidnameTrue(str2,option);
+    	else if(str1.contains("cl")) return isValidclTrue(str2);
+    	else if(str1.contains("phoneNum")) return isValidphoneNumTrue(str2,option);
+    	else if(str1.contains("birth")) return isValidbirthdaytTrue(str2,option);
+    	else if(str1.contains("certi")) return isValidcertiTrue(str2);
+    	return false;
+    }
+    
+    
+    public static boolean isValidOptionTrue(String str) {
+    	String regx = "^\\-[dflmpy]";
+    	return Pattern.matches(regx, str);
+    }
+    
+    public static COMMAND getcommand(String str) {
+    	if(isValidCommandTrue(str)) {
+    		return COMMAND.valueOf(str);
+    	}
+    	return null; // valid 오류 시 리턴 어떻게 할건지?
+    }
+
+    public static List<String> getArgument(String []str) {
+    	List<String> Argument = new ArrayList();
+
+    	if(str[0].contains("ADD") && isValidAddArgsTrue(str)) {
+        	for(int index = 4 ; index < str.length ; index++) {
+        			Argument.add(str[index]);
+        	}
+    	}
+    	else if((str[0].contains("DEL") || str[0].contains("SCH"))){
+    		if(isValidFirstArgsTrue(str[4])) 
+    			Argument.add(str[4]);
+    		if(isValidSecondArgsTrue(str[4],str[5],str[2])) 
+    			Argument.add(str[5]);
+    	}
+    	else if(str[0].contains("MOD")) {
+
+    		if(isValidFirstArgsTrue(str[4])) 
+    			Argument.add(str[4]);
+    		if(isValidSecondArgsTrue(str[4],str[5],str[2])) 
+    			Argument.add(str[5]);
+    		if(isValidFirstArgsTrue(str[6])) 
+    			Argument.add(str[6]);
+    		if(isValidSecondArgsTrue(str[6],str[7],str[2])) 
+    			Argument.add(str[7]);
+    	}
+    	return Argument;
+    }
+    
+    public static List<String> getOption(String []str) {
+	   	 List<String> Argument = new ArrayList();
+		 
+	   	for(int index = 1 ; index < 3 ; index++) {
```

Comment:
```suggestion
	   	for(int index = 1 ; index <= 3 ; index++) {
```
현재 옵션 3번이 비어있지만 공백으로 같이 입력되지 않나요? 아직 사용하지 않음 이여서 빼신건지요